core/src/integrationTest/java/org/lflang/tests/RuntimeTest.java

```diff
@@ -209,8 +221,33 @@ public void runEnclaveTests() {
     runTestsForTargets(
         Message.DESC_ENCLAVE,
         TestCategory.ENCLAVE::equals,
+        Transformers::noChanges,
         Configurators::noChanges,
         TestLevel.EXECUTION,
         false);
   }
+
+  /** Given a test category, return true if it is compatible with single-threaded execution. */
+  public static boolean compatibleWithThreadingOff(TestCategory category) {
+
+    // CONCURRENT, FEDERATED, DOCKER_FEDERATED, DOCKER
+    // are not compatible with single-threaded execution.
+    // ARDUINO and ZEPHYR have their own test suites, so we don't need to rerun.
+    boolean excluded =
+        category == TestCategory.CONCURRENT
+            || category == TestCategory.SERIALIZATION
+            || category == TestCategory.FEDERATED
+            || category == TestCategory.DOCKER_FEDERATED
+            || category == TestCategory.DOCKER
+            || category == TestCategory.ENCLAVE
+            || category == TestCategory.ARDUINO
+            || category == TestCategory.VERIFIER
+            || category == TestCategory.ZEPHYR_UNTHREADED
+            || category == TestCategory.ZEPHYR_BOARDS
+            || category == TestCategory.ZEPHYR_THREADED;
+
+    // SERIALIZATION and TARGET tests are excluded on Windows.
```

Comment:
Looks like `SERIALIZATION` is excluded without checking the platform. So there is a small mismatch between the comment and the implementation.

Reply:
Yep. I'll fix that later. No need to run CI again for a comment change. Thanks!